### PR TITLE
add test to AppTextField #122

### PR DESCRIPTION
**Pull request by Mygal-Alex**
image

image

### CODE REVIEW

**Review by AndriiNasadiuk — APPROVED** [review]

**Review by marianamartsy — APPROVED** [review]
👍

**Review by LeonidShamarin — APPROVED** [review]

**Review by ArtemHolikov — APPROVED** [review]
Correct

**Comment by vlad-khrychov** [comment]
Pleas add `close #122` to description

File: tests/unit/components/app-text-field/AppTextField.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    expect(input).toBeInTheDocument()
+    expect(errorTooltip).toBeInTheDocument()
+  })
+
+  it('should render without error message', () => {
+    render(<AppTextField />)
+
+    const input = screen.getByRole('textbox')
+    const errorText = screen.queryByText(/TEST_ERROR/)
```
Comment (vlad-khrychov) [inline tests/unit/components/app-text-field/AppTextField.spec.jsx:21]: Here we need to use `data-testId` or delete this and 25 row

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

**Pull request merged**